datastreams/transport.go

```diff
@@ -61,6 +61,9 @@ func newHTTPTransport(addr string, site string, apiKey string, client *http.Clie
 		"Content-Type":                  "application/msgpack",
 		"Content-Encoding":              "gzip",
 	}
+	if cid := ContainerID(); cid != "" {
+		defaultHeaders["Datadog-Container-ID"] = cid
```

Comment:
Maybe something like DD-K8S-CONTAINER-ID?

Reply:
It's already set: https://github.com/DataDog/datadog-agent/blob/4e2e857715027ab81641046a2555dc650fb1660c/pkg/trace/api/internal/header/headers.go#L16
I used the same as APM is using.